Core/include/Acts/Seeding/SeedFinderUtils.ipp

```diff
@@ -178,56 +178,61 @@ inline bool xyzCoordinateCheck(
 
   std::size_t index = sp.index();
 
-  const float& topHalfStripLength = spacePointData.getTopHalfStripLength(index);
-  const float& bottomHalfStripLength =
-      spacePointData.getBottomHalfStripLength(index);
-  const Acts::Vector3& topStripDirection =
-      spacePointData.getTopStripDirection(index);
-  const Acts::Vector3& bottomStripDirection =
-      spacePointData.getBottomStripDirection(index);
+  //  const float& topHalfStripLength =
+  //  spacePointData.getTopHalfStripLength(index); const float&
+  //  bottomHalfStripLength =
+  //      spacePointData.getBottomHalfStripLength(index);
+  //  const Acts::Vector3& topStripDirection =
+  //      spacePointData.getTopStripDirection(index);
+  //  const Acts::Vector3& bottomStripDirection =
+  //      spacePointData.getBottomStripDirection(index);
+
+  // prepare variables
+  const Acts::Vector3& topStripVector = spacePointData.getTopStripVector(index);
```

Comment:
`Acts::Vector3` uses `double`. So we may want to remain to `double` here and in the other places. We can then evaluate what is the effect of moving from `double` to `float` but I'd keep that for another PR